arch/arm/core/cortex_m/mpu/arm_mpu.c

```diff
@@ -207,7 +207,7 @@ void arm_core_mpu_configure_user_context(struct k_thread *thread)
 	 * it is essential to include this size increase, to avoid
 	 * MPU faults.
 	 */
-	size += thread->stack_info.start - thread->stack_obj;
+	size += thread->stack_info.start - (u32_t)thread->stack_obj;
```

Comment:
@galak nice! We had fixed that downstream, but deferred from fixing this upstream until we actually get a v8m board with USERSPACE support :) No objection of fixing it under this PR.

Could you, perhaps, replace `(u32_t)thread->stack_obj` with `base`?

Reply:
Done

Reply:
yeah we'll do that